commands/Create.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs-extra');
+
+Promise.all(
+    [
+        fs.outputFile(`./rules/${ process.argv[2] }/README.md`, `#### 📍 <rule-name>\r\n\r\n_Please give a short description of the rule_`)
```

Comment:
I'd probably create .stub files that you copy the content out of rather than having it all inline. Will make it easier to read and modify. A file structure such as:

```
commands/create-rule/index.js
commands/create-rule/stubs/README.md
commands/create-rule/stubs/incorrect.css
commands/create-rule/stubs/correct.css
```